src/shared/bootspec.c

```diff
@@ -437,12 +435,9 @@ static int find_sections(
 
         _cleanup_free_ struct PeSectionHeader *sections = NULL;
         _cleanup_free_ char *osrelease = NULL, *cmdline = NULL;
-        size_t i, n_sections;
-        struct DosFileHeader dos;
-        struct PeHeader pe;
-        uint64_t start;
         ssize_t n;
 
+        struct DosFileHeader dos;
```

Comment:
hu?

Reply:
What do you mean? It's a variable declaration that was moved down to the point where it is used so that the reader doesn't have to scan the text up and down to see what is happening.

Reply:
Please look at the whole commit: other variables that are defined and used in the same fashion are moved a bit more. This is the first one, so it's move just a bit down.